src/query/range_query_ip_fastfield.rs

```diff
@@ -589,7 +649,139 @@ mod bench {
             let start = Ipv6Addr::from_u128(0);
             let end = Ipv6Addr::from_u128(90 * 1000);
 
-            excute_query(start, end, "AND id:veryfew", &index)
+            excute_query("ip", start, end, "AND id:veryfew", &index)
+        });
+    }
+
+    #[bench]
+    fn bench_ip_range_hit_90_percent_multi(bench: &mut Bencher) {
+        let index = get_index_0_to_100();
```

Comment:
can we seed those, and add an assert on the number of docs returned?

The bench is strongly relying on the non documented quirky spec of `get_index_0_to_100`.
I'd like to make sure we don't break some bench by modifying the spec of that function.

Reply:
I moved the selectors to functions and added a comment